Flaky test

The `test_delete_all_tasks` in `tests/client/test_client_task_meilisearch.py` is a flaky test. Frequently it fails with `AssertionError: assert 5 == 1`, then rerunning it will pass.

--- a/tests/client/test_client_task_meilisearch.py
+++ b/tests/client/test_client_task_meilisearch.py
@@ -151,7 +151,7 @@ def test_delete_tasks_by_uid(client, empty_index, small_movies):
     assert f"uids={task_addition.task_uid}" in task["details"]["originalFilter"]
 
 
-def test_delete_all_tasks(client):
+def test_delete_tasks_by_filter(client):
     tasks_before = client.get_tasks()
     task = client.delete_tasks({"statuses": ["succeeded", "failed", "canceled"]})
     client.wait_for_task(task.task_uid)

